docs/releases.rst

```diff
@@ -3,6 +3,133 @@
 Release notes
 =============
 
+Version 1.3
+-----------
+
+Version 1.3 adds custom bootstap possibility, significantly improves support for pg_rewind, enhances the
+synchronous mode support, adds configuration editing to patronictl and implements watchdog support on Linux.
+In addition, this is the first version to work correctly with PostgreSQL 10.
+
+**Custom bootstrap**
+
+- Make the process of bootstrapping the cluster configurable (Alexander Kukushkin)
+
+  Allow custom bootstrap scripts instead of ``initdb`` when initializing the very first node in the cluster.
+  The bootstrap command receives the name of the cluster and the path to the data directory. The resulting cluster can
+  be configured to start in recovery, making it possible to bootstrap a new cluster from the point in the timeline
+  of an existing one. Refer to the :ref:`documentaton page<custom_bootstap>` for more detailed description of this feature.
+
+**Smarter pg_rewind support**
+
+-  Decide on whether to run pg_rewind by looking at the timeline differences from the current master (Alexander)
+
+   Previously, Patroni had a fixed set of conditions to trigger pg_rewind, namely when starting a former master, when
+   doing a switchover to the designated node that is not a master or when there is a replica with the nofailover tag,
+   basically those cases when there is a chance that a replica may be ahead of the new master. In some cases, pg_rewind
+   did nothing, in some other ones it was not running when necessary. Instead of relying on this limited list of rules
+   this change makes Patroni compare the master and the replica WAL positions (using the streaming replication protocol)
+   in order to reliably decide if the replica to join is ahead of the new master.
+
+**Synchronous replication mode strict**
+
+-  Enhance synchronous replication support by adding the strict mode (James Sewell, Alexander)
+
+   When the new configuration option ``synchronous_mode_strict`` is set, Patroni will not let the master accept write
+   requests if there are no replicas attached . This option provides the guarantee that no writes will be lost because
```

Comment:
The non-strict synchronous mode is "no writes will be lost due to automatic failover". The strict mode is "all writes are two node durable", meaning that a catastrophic loss of any one node (node, including storage, is gone and is not coming back) will not lead to lost writes.

The situation where it matters is when the sync standby goes down for a bit and while the standby is down and a new one hasn't been chosen the master completely explodes.